src/scss/components/_inputs.scss

```
@@ -144,6 +144,30 @@ $input-select-margin-right: 1;
   }
 }
 
+.usa-checkbox__label-illustrated,
+.usa-radio__label-illustrated {
+  @include at-media('tablet') {
+    display: flex;
+  }
+  & .usa-radio__image,
```

It's safe to leave out the `&`, since it means the same thing anyways.

```suggestion
  .usa-radio__image,
```